src/app/pages/user/dialogs/update/update-user.dialog.html

```diff
@@ -56,17 +56,16 @@ <h3 mat-dialog-title>עדכון פרטים אישיים</h3>
           <div class="msw-form-row">
             <mat-form-field color="primary">
               <mat-select placeholder="כיתה" >
-                <mat-option [value]="">בחר</mat-option>            
+                <mat-option [value]="">בחר</mat-option>
               </mat-select>
             </mat-form-field>
           </div>
         </div>
-      </form>
-    </mat-dialog-content>
-    <div mat-dialog-actions>
-      <button  mat-button class="msw-button-cancel" tabindex="-1" (click)="close()">ביטול</button>
-      <!-- <button [type]="submit" class="mat-raised-button mat-primary" (click)="updateDetails()" [disabled]="!formControl.valid" >עדכן משתמש</button> -->
-      <button  mat-button class="msw-button-update" (click)="updateDetails()" [disabled]="!formControl.valid" >עדכן</button>
-    </div>
+      </mat-dialog-content>
+      <div mat-dialog-actions>
+        <button  mat-button class="msw-button-cancel" tabindex="-1" (click)="close()">ביטול</button>
+         <!-- <button [type]="submit" class="mat-raised-button mat-primary" (click)="updateDetails()" [disabled]="!formControl.valid" >עדכן משתמש</button> -->
+        <button type="submit" mat-button class="msw-button-update" [disabled]="!formControl.valid" >עדכן</button>
```

Comment:
(click)="updateDetails()"  is missing in that line. How do the button know what function to activate?

Reply:
Hi @eran1205,
This button is now the form's "official" submit button.
Both Enter keypress and this button click triggers the "updateDetails()" handler